### PR TITLE
PS-1065: API implementation process updates

### PR DESCRIPTION
**Pull request by andyartz**

### CODE REVIEW

File: implementation/api-integration.mdx [hunk 1]
```diff
@@ -14,4 +14,40 @@ clinical information about each patient as possible (ex: diagnoses, medications)
...
 as part of their reciprocity requirements.
 
-More information on the Connective Health Clinical Data Processing API can be found [here](/scheduled-appointments/patient-data-processing).
+The Connective Health Clinical Data Processing API specification can be found [here](/scheduled-appointments/patient-data-processing).
+
+## API Implementation Process
+
+### Staging Environment Testing
+
+Before connecting to the production API, integration is tested in the Staging environment.
```
Comment (jjathman-ch) [inline implementation/api-integration.mdx:23]: Capitalization of production and staging isn't consistent. I assume they should be?

File: implementation/api-integration.mdx [hunk 2]
```diff
@@ -14,4 +14,40 @@ clinical information about each patient as possible (ex: diagnoses, medications)
 This information is shared back with the various health information exchange networks
 as part of their reciprocity requirements.
 
-More information on the Connective Health Clinical Data Processing API can be found [here](/scheduled-appointments/patient-data-processing).
+The Connective Health Clinical Data Processing API specification can be found [here](/scheduled-appointments/patient-data-processing).
+
+## API Implementation Process
+
+### Staging Environment Testing
```
Comment (jjathman-ch) [inline implementation/api-integration.mdx:21]: Should be have a big callout that only fake data is allowed within staging?

File: implementation/api-integration.mdx [hunk 3]
```diff
@@ -14,4 +14,40 @@ clinical information about each patient as possible (ex: diagnoses, medications)
...
+It is okay to use any non-PHI sample patient information for testing purposes, however only sample patients provided by Connective Health will receive responses. Because Connective Health’s staging environment interacts with live networks and other staging environments across outside EHRs, certain patients may or may not receive a response. To maximize the chances of a successful response it is recommended that API requests are sent for all patients using all provided information.
+
+### Production Environment Testing
+
+Once finished with staging, the next step is production testing.
+
+1. Connective Health sends production information package to customer:
+    * Production API key
+    * Production test endpoint URL
+1. Customer sends 10 live patient requests to Connective Health’s production test endpoint. These patients are not processed; they are simply for the purposes of validation.
```
Comment (jjathman-ch) [inline implementation/api-integration.mdx:47]:
```suggestion
1. Customer sends between 5 and 10 real patient requests to Connective Health’s production test endpoint. These patients are not processed; they are simply for the purposes of validation. All data normally included for a production request should also be sent to the test endpoint.
```

**Review by jjathman-ch — APPROVED** [review]

File: implementation/api-integration.mdx (hunk 2 above)
Comment (andyartz) [reply implementation/api-integration.mdx:21]: see if you like

File: implementation/api-integration.mdx (hunk 1 above)
Comment (andyartz) [reply implementation/api-integration.mdx:23]: yeah, let's keep it lower-case except in titles or beginning of sentences.

**Review by jjathman-ch — APPROVED** [review]

**Pull request merged**